app/controllers/base_controller.rb

```diff
@@ -1,3 +1,8 @@
 class BaseController < ActionController::Base
   layout 'application'
+
+  def current_legal_aid_application
+    return unless session[:current_application_ref].present?
```

Comment:
If either the `session[:current_application_ref]` or the application itself don't exist, isn't the expected behaviour the same? For the user to be shown an error page? 

I would say, if the application existence is a pre-condition to access these page, this should be defined as a `before_action` to enforce its existence or redirect the user to an error page.

Reply:
This method just gets a `legal_aid_application`. I was expecting the specific controller to decide on how to deal with it's absence. Also as the behaviour through the Citizen Journey didn't seem to be finalised I didn't want to put too many restrictions on its use.